### DISCUSSION TITLE
Dark mode - text color

### DISCUSSION THREAD

**Discussion by honey-trap** [mastodon/mastodon#17253] [General]
Many users prefer dark mode opposed to light mode.
The white text on soft pink background is quite difficult to view.
Recommend text color change from white to other.

<img width="1436" alt="Screen Shot 2022-01-05 at 4 25 00 PM" src="https://user-images.githubusercontent.com/97202868/148320154-a36bb725-58ac-4361-9e92-2db2ac78a859.png">
)

**Comment by ClearlyClaire** [comment]
This seems like an issue with your instance or your browser, the background on this page is supposed to be dark (the same color as on the left).

**Reply by honey-trap** [reply]
I use Brave browser. No problem then. Thanks!
